bulk_grades/views.py

```diff
@@ -91,12 +99,10 @@ def post(self, request, course_id, *args, **kwargs):
             data = self.processor.status()
             data['error_messages'] = []
             for error_message in self.processor.error_messages:
+                is_plural = 's' if len(line_numbers) > 1 else ''
                 line_numbers = ', '.join(str(line_number+1) for line_number in self.processor.error_messages[error_message])
-                new_message = '{error_message} on line{is_plural} {line_numbers}'.format(
-                                    error_message=error_message,
-                                    is_plural='s' if len(line_numbers) > 1 else '',
-                                    line_numbers=line_numbers
-                                )
+                new_message = f'{error_message} on line{is_plural} {line_numbers}'
```

Comment:
I meant to do this for awhile, but is it okay to ask you to squeeze [AU-66](https://openedx.atlassian.net/browse/AU-66) in here? It was my mistake and I haven't got to do it. Just update it to `f'{error_message} (on line{is_plural} {line_numbers})'`

Reply:
Sure thing 👍